KeyboardManagerDemo/KeyboardManagerDemo/AppDelegate.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AppDelegate.swift
+//  KeyboardManagerDemo
+//
+//  Created by Alexey Korolev on 15.04.17.
+//  Copyright © 2017 Alexey Korolev. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+    func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    func applicationWillResignActive(_: UIApplication) {
+        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
+        // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/alphatroya/KeyboardManager/blob/b88ef862088cfe316885e971372c9fbbf52f8dcc/KeyboardManagerDemo/KeyboardManagerDemo/AppDelegate.swift#L23"></span>Line should be 140 characters or less: currently 159 characters</td>
    </tr>
  </tbody>
</table>